pkgs/by-name/gv/gvmd/package.nix

```diff
@@ -0,0 +1,114 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchurl
+, cmake
+, gvm-libs
+, pkg-config
+, gnutls
+, xml-tooling-c
+, glib
+, doxygen
+, git
+, libbsd
+, libical
+, libpkgconf
+, gpgme
+, mount
+, libidn2
+, graphviz
+, icu
+, p11-kit
+, pcre2
+, libxslt
+, libtasn1
+, postgresql_16
```

Comment:
Why import a specific postgresql version here?

Reply:
with postgresql_16, the highest version will be used for build process while posgresql uses v15.6.

Reply:
Sure, but does the package explicitly depend on 16?

Reply:
no, prerequisite: * PostgreSQL database >= 9.6

Reply:
Okay, so please use the normal `postgresql` attribute, users can overwrite it themselves if they prefer to use a newer database!

Reply:
Sure I will do.